Week1/homework/index.html

```diff
@@ -1 +1,23 @@
-<!-- replace this with your HTML content -->
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <nav>
```

Comment:
I like your choice of using semantic HTML tags i.e `header`, `nav` and so on. You can also look up `main` tag where you can insert the main content of your app which in this case is the books 🙂

Reply:
I changes div with html<main> element!